datastore/key/namespace.nim

```diff
@@ -19,32 +18,26 @@ type
 
 func init*(T: type Namespace, field, value: string): ?!T =
   if value.contains(Delimiter):
-      return failure (&"value string must not contain Delimiter '{Delimiter}'")
-        .catch.expect("should not fail")
+      return failure ("value string must not contain Delimiter " & Delimiter)
 
   if value.contains(Separator):
-    return failure (&"value string must not contain Separator {Separator}")
-      .catch.expect("should not fail")
+    return failure ("value string must not contain Separator " & Separator)
 
   if field.contains(Delimiter):
-    return failure (&"field string must not contain Delimiter {Delimiter}")
-      .catch.expect("should not fail")
+    return failure ("field string must not contain Delimiter " & Delimiter)
 
   if field.contains(Separator):
-    return failure (&"field string must not contain Separator {Separator}")
-      .catch.expect("should not fail")
+    return failure ("field string must not contain Separator " & Separator)
```

Comment:
Just out of curiosity, why did you decide to replace string interpolation with concatenation?

Reply:
`&`/`{}` raise `ValueError`, which is why there was the weird `catch.expect` attached there, which just feels cumbersome for very little gain, IMO...

Reply:
Thanks for the explanation 👌